src-cpp/rdkafkacpp.h

```diff
@@ -2357,6 +2357,22 @@ class RD_EXPORT KafkaConsumer : public virtual Handle {
    *         RdKafka::KafkaConsumer::assign() */
   virtual ErrorCode assignment (std::vector<RdKafka::TopicPartition*> &partitions) = 0;
 
+
+  /** @brief Check whether the consumer considers the current assignment to
+   *         have been lost involuntarily. This method is only applicable for
+   *         use with a subscribing consumer. Assignments are revoked
+   *         immediately when determined to have been lost, so this method is
+   *         only useful within a rebalance callback.
```

Comment:
Should also say what this means for the application, like not able to commit offsets, right?

Reply:
good idea